llvm/include/llvm/Frontend/OpenMP/OMPKinds.def

```diff
@@ -458,6 +458,8 @@ __OMP_RTL(__tgt_push_mapper_component, false, Void, VoidPtr, VoidPtr, VoidPtr,
 __OMP_RTL(__kmpc_task_allow_completion_event, false, VoidPtr, IdentPtr,
           /* Int */ Int32, /* kmp_task_t */ VoidPtr)
 
+__OMP_RTL(ompx_dump_mapping_tables, false, Void, )
```

Comment:
Why do we need this here? Are we emitting calls from `clang`?

Reply:
You're right we don't need it. I checked other openmp api extensions to see what needed to be done to support it but as you pointed out this isn't necessary.